commands/audit/scarunner.go

```diff
@@ -57,8 +57,8 @@ func buildDepTreeAndRunScaScan(auditParallelRunner *utils.SecurityParallelRunner
 		log.Debug("Skipping SCA scan as requested by input...")
 		return
 	}
-	if auditParams.configProfile != nil {
-		log.Debug("Skipping SCA scan as a configuration profile is being utilized and currently only Secrets and Sast scanners are supported when utilizing a configuration profile")
+	if auditParams.configProfile != nil && !auditParams.configProfile.Modules[0].ScanConfig.EnableScaScan {
```

Comment:
I'd add some validations here, to avoid a nil pointer panic (`Modules` array mostly

Reply:
There cannot be a profile without a module, and if there is i'd rather getting Panic and know about it since I dont want to validate a wrong behaviour